### PR TITLE
Various file edit for bs4 installation along with readme update to include webdriver installation and firefox fix

### PR DESCRIPTION
**Pull request by fakeid30**
The files had wrong instruction in a block , cleaned them up.

### CODE REVIEW

**Comment by fakeid30** [comment]
Fixed the firefox issue and added webdriver python pip module.

**Comment by fakeid30** [comment]
undid the previous fix and this time fixed it for real. Marionette false is not working past Firefox version 46 , the fix is taken from https://stackoverflow.com/a/59015416

File: udemy_enroller_firefox.py [hunk 1]
```diff
@@ -28,7 +28,8 @@
 
 email, password, zipcode = f.readline().rstrip('\n'), f.readline().rstrip('\n'), f.readline().rstrip('\n')
 
-driver = webdriver.Firefox(executable_path=GeckoDriverManager().install())
+
+driver = webdriver.Firefox()
```
Comment (fakeid30) [inline udemy_enroller_firefox.py:32]: you need to manually install geckodriver for now, i think.

**Review by fakeid30 — COMMENTED** [review]
@aapatre what do you think?

**Comment by aapatre** [comment]
Hi, thanks a lot for fixing the Firefox issue. I'm accepting pull request but can you please 'uncommit' the comment changes? Cause I totally removed those and added new ones. :)

**Comment by fakeid30** [comment]
> 
> 
> Hi, thanks a lot for fixing the Firefox issue. I'm accepting pull request but can you please 'uncommit' the comment changes? Cause I totally removed those and added new ones. :)

i am not sure i understand what you mean by that . Do I discard the changes to fix bs4 installation?

**Comment by fakeid30** [comment]
@aapatre is that ok now ?

**Review by aapatre — APPROVED** [review]

**Review by aapatre — APPROVED** [review]

**Review by aapatre — APPROVED** [review]

**Review by aapatre — APPROVED** [review]

**Comment by aapatre** [comment]
The pull request removed header comments from all the files, you can send me another pull request to update the docs to explain how to make it work on Firefox and please make changes to the Firefox code only.

**Pull request merged**